src/DGtal/geometry/doc/moduleHull2D.dox

```diff
@@ -0,0 +1,307 @@
+/**
+ * @file moduleHull2D.dox
+ * @author Tristan Roussillon (\c tristan.roussillon@liris.cnrs.fr )
+ * Laboratoire d'InfoRmatique en Image et Systèmes d'information - LIRIS (CNRS, UMR 5205), CNRS, France
+ *
+ * @date 2013/12/19
+ *
+ * Documentation file for feature moduleHull2D
+ *
+ * This file is part of the DGtal library.
+ */
+
+/*
+ * Useful to avoid writing DGtal:: in front of every class.
+ * Do not forget to add an entry in src/DGtal/base/Config.h.in !
+ */
+namespace DGtal {
+//----------------------------------------
+/*!
+@page moduleHull2D Convex hull and alpha-shape in the plane
+@writers Tristan Roussillon
+[TOC]
+
+In this part of the manual, we describe how to compute 
+well-known geometric structures such as the convex hull 
+or the alpha-shape of a 2D point set.  The proposed 
+implementations provide exact results as long as the underlying
+geometric predicates are robust (see @ref moduleGeometricPredicates 
+for more details about geometric predicates).
+
+Related examples are exampleConvexHull2D.cpp and exampleAlphaShape.cpp. 
+
+\section sectmoduleHull2D1 Convex hull
+
+Three convex hull algorithms are available: 
+- functions::Hull2D::andrewConvexHullAlgorithm [Andrew, 1979 : \cite Andrew1979]
+- functions::Hull2D::grahamConvexHullAlgorithm [Graham, 1972 : \cite Graham1972]
+- functions::Hull2D::melkmanConvexHullAlgorithm [Melkman, 1987 : \cite Melkman1987]
+
+They both use the basic procedures Hull2D::buildHullWithStack
+(the object having a stack interface is passed by reference) 
+or Hull2D::buildHullWithAdaptedStack (the object having a stack 
+interface is passed by copy). These procedures retrieves the 
+extremal vertices of a weakly externally visible polygon (WEVP) 
```

Comment:
Ok the def is at the end of the paragraph. Maybe it would help to reverse the paragraph statements, starts from easy things (WEVP and algorithms def) and then give the interface (buildHullWithStack or buildHullWithAdaptedStack)